module/storage-logzio/src/main/resources/zipkin-server-logzio.yml

```diff
@@ -1,14 +1,17 @@
 # When enabled, this allows shorter env properties (ex -Dspring.profiles.active=logzio)
+spring:
+  main:
+    # ZipkinLogzioStorageModule overrides the bean named 'storage'
+    allow-bean-definition-overriding: true
 zipkin:
+  internal:
+    module:
+      logzio: zipkin.module.storage.logzio.ZipkinLogzioStorageModule
   storage:
-    strict-trace-id: ${STRICT_TRACE_ID:true}
```

Comment:
this was redundant to the main file here https://github.com/openzipkin/zipkin/blob/master/zipkin-server/src/main/resources/zipkin-server-shared.yml#L91